api/src/test/java/org/pmiops/workbench/initialcredits/InitialCreditsServiceTest.java

```diff
@@ -1098,8 +1098,11 @@ public void test_extendInitialCreditsExpiration_notYetExtendedWithinWindow() {
                         .setExtensionTime(null)
                         .setBypassed(false)));
 
-    initialCreditsService.extendInitialCreditsExpiration(user);
-    DbUserInitialCreditsExpiration actualExpirationRecord = user.getUserInitialCreditsExpiration();
+    DbUserInitialCreditsExpiration actualExpirationRecord =
```

Comment:
Adjusted test to take advantage of the fact that the service is now returning an updated DbUSer object.